backend/integrations/discord/cogs.py

```
@@ -391,3 +399,124 @@ async def list_indexed_repos(self, interaction: discord.Interaction):
+            auth_url = None
+
+            if show_oauth_button:
+                # Create verification session
+                session_id = await create_verification_session(str(user.id))
+                if not session_id:
+                    try:
+                        await user.send("I couldn't start verification right now. You can use /verify_github anytime.")
+                        await user.send(build_encourage_verification_message(reminder_count=1))
+                        await user.send(embed=build_final_handoff_embed())
+                    except Exception:
+                        pass
```

Ruff (0.14.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)